src/harmony/Text/Text.module.css

```diff
@@ -1,7 +1,4 @@
-.Text {
-    font-size: 1rem;
-    color: var(--text-primary);
-}
+.Text {}
```

Comment:
Also, if color is removed from the base component it should be set somewhere at the root of the app

Reply:
I think. Maybe it should be done in project, not here